charts/rollups-node/ci/test-values.yaml.tpl

```diff
@@ -22,7 +26,7 @@ validator:
     CARTESI_FEATURE_DISABLE_MACHINE_HASH_CHECK: "true"
     CARTESI_SNAPSHOT_DIR: "/usr/share/cartesi/snapshot"
   initContainers:
-    - image: "docker.io/cartesi/dapp:echo-python-0.16.0-server"
+    - image: "{{ .Values.validator.application.image.repository }}:{{ .Values.validator.application.image.tag }}"
```

Comment:
we should reuse the helper method at https://github.com/cartesi/helm-charts/blob/main/charts/rollups-node/templates/_helpers.tpl#L107-L121

or create a wrapper like https://github.com/cartesi/helm-charts/blob/main/charts/rollups-node/templates/_helpers.tpl#L137-L142

so we could use `{{ include "application.image" . }}` at the end

Reply:
the include function should typically be used within template files rather than directly within the YAML structure of `values.yaml`.

> so we could use `{{ include "application.image" . }}` at the end

Reply:
I think, we can use this approach to make it simpler : 

```
validator:
  application:
    image: cartesi/dapp:echo-python-0.16.0-server

```

And then fetch the value: 

```
validator:
  initContainers:
    - image: "{{ .Values.validator.application.image }}"
```

Reply:
OR we can remove the `- image` section from `initcontainer` of values.yaml to the deployment template. 
I'll send a fixup for it.

Reply:
https://github.com/cartesi/helm-charts/pull/83/commits/3179ad62a6b1565498d578b9ddd22e494e36e994

https://github.com/cartesi/helm-charts/pull/83/commits/d5150da3978203c6e06f31641e7ff05083e5cf3f